blueprints/viet_kwik_e_mart/config.py

```diff
@@ -0,0 +1,105 @@
+LANGUAGE_CONFIG = {"language": "vi", "locale": "vi_VI"}
+
+NLP_CONFIG = {
+    "resolve_entities_using_nbest_transcripts": ["store_info.get_store_hours"]
+}
+
+DOMAIN_CLASSIFIER_CONFIG = {
+    "model_type": "text",
+    "model_settings": {"classifier_type": "logreg",},
+    "param_selection": {
+        "type": "k-fold",
+        "k": 10,
+        "grid": {"fit_intercept": [True, False], "C": [10, 100, 1000, 10000, 100000]},
+    },
+    "features": {
+        "bag-of-words": {"lengths": [1],},
+        "freq": {"bins": 5},
+        "in-gaz": {},
+        "exact": {},
+        "enable-stemming": False,
+    },
+}
+
+INTENT_CLASSIFIER_CONFIG = {
+    "model_type": "text",
+    "model_settings": {"classifier_type": "logreg"},
+    "param_selection": {
+        "type": "k-fold",
+        "k": 10,
+        "grid": {
+            "fit_intercept": [True, False],
+            "C": [0.01, 1, 100, 10000, 1000000],
+            "class_bias": [1, 0.7, 0.3, 0],
+        },
+    },
+    "features": {
+        "bag-of-words": {"lengths": [1]},
+        "in-gaz": {},
+        "freq": {"bins": 5},
+        "exact": {},
+        "length": {},
+        "enable-stemming": False,
+    },
+}
+
+ENTITY_RECOGNIZER_CONFIG = {
+    "model_type": "tagger",
+    "label_type": "entities",
+    "model_settings": {
+        "classifier_type": "memm",
+        "tag_scheme": "IOB",
+        "feature_scaler": "max-abs",
+    },
+    "param_selection": {
+        "type": "k-fold",
+        "k": 5,
+        "scoring": "accuracy",
+        "grid": {
+            "penalty": ["l1", "l2"],
+            "C": [0.01, 1, 100, 10000, 1000000, 100000000],
+        },
+    },
+    "features": {
+        "bag-of-words-seq": {
+            "ngram_lengths_to_start_positions": {
+                1: [-2, -1, 0, 1, 2],
+                2: [-2, -1, 0, 1],
+            }
+        },
+        "in-gaz-span-seq": {},
+        "sys-candidates-seq": {"start_positions": [-1, 0, 1]},
+        "enable-stemming": False,
+    },
+}
+
+
+TEST_ENTITY_RECOGNIZER_CONFIG = {
+    "model_type": "tagger",
+    "label_type": "entities",
+    "model_settings": {
+        "classifier_type": "memm",
+        "tag_scheme": "IOB",
+        "feature_scaler": "max-abs",
+    },
+    "params": {"penalty": "l1", "C": 50},
+    "features": {
+        "bag-of-words-seq": {
+            "ngram_lengths_to_start_positions": {
+                1: [-2, -1, 0, 1, 2],
+                2: [-2, -1, 0, 1],
+            }
+        },
+        "in-gaz-span-seq": {},
+        "sys-candidates-seq": {"start_positions": [-1, 0, 1]},
+        "enable-stemming": False,
+    },
+    "train_label_set": "testtrain.*\.txt",  # noqa: W605
```

Comment:
this is why there is no entity because the pattern is incorrect and we can't find any files for training